ui/src/app/job-list/table/table.component.spec.ts

```diff
@@ -204,10 +208,24 @@ describe('JobsTableComponent', () => {
       .toEqual(jobs[0].labels['status-detail']);
   }));
 
+  it('should not display editable field for job label if config has not explicitly said it is editable', async(() => {
+    fixture.detectChanges();
+    let de: DebugElement = fixture.debugElement;
+    expect(de.queryAll(By.css('.cdk-column-labels-status-detail .edit-field')).length)
+      .toEqual(0);
+  }));
+
+  it('should display editable field for job label if config has explicitly said it is editable', async(() => {
+    fixture.detectChanges();
+    let de: DebugElement = fixture.debugElement;
+    expect(de.queryAll(By.css('.cdk-column-labels-comment .edit-field')).length)
+      .toEqual(de.queryAll(By.css('.cdk-column-labels-comment')).length - 1);
```

Comment:
I don't think I follow this logic. Why is this not equal to 1?

Reply:
There's one comment edit field per job.

Reply:
Ah thanks - comment?